tests/test_estimation.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import pymc3 as pm
+import theano.tensor as tt
+
+from pymc3_hmm.distributions import DiscreteMarkovChain, PoissonZeroProcess
+from pymc3_hmm.step_methods import FFBSStep
+from tests.utils import simulate_poiszero_hmm
+
+
+# the way we use this initial probability state vector
+def test_gamma_0_estimation():  # annotate this test with pytest
+
+    np.random.seed(2032)
+
+    poiszero_sim, _ = simulate_poiszero_hmm(
+        30, 150, pi_0=[0.5, 0.5]
+    )  # test [.01, .99], [.99, .01]
+    y_test = poiszero_sim["Y_t"]
+
+    with pm.Model() as test_model:
+        p_0_rv = np.array([0.5, 0.5])
+        p_1_rv = np.array([0.5, 0.5])
+
+        P_tt = tt.stack([p_0_rv, p_1_rv])  # transition matrix
+        P_rv = pm.Deterministic("P_tt", tt.shape_padleft(P_tt))
+
+        # how well is this estimated?
+        pi_0_rv = pm.Dirichlet("pi_0", np.r_[1, 1])  # "flat" prior for initial states
+
+        S_rv = DiscreteMarkovChain("S_t", P_rv, pi_0_rv, shape=y_test.shape[0])
+
+        PoissonZeroProcess("Y_t", 150, S_rv, observed=y_test)
```

Comment:
Since we only want to test the initial state estimation, let's simplify this model so that nothing else interferes with that.  Instead of using a Poisson, let's start by using a mixture with two `pm.Constant` distributions (for the simulated "true" data, as well).

If we use `pm.Constant`s at the same values as the state values (i.e. `0` and `1`), then we can effectively bypass the emission distributions and only deal with the state sequence and initial states.  The log-likelihoods end up being much simpler and easier to reason about on the fly.